### PR TITLE
Build container images for releases

### PR DESCRIPTION
**Pull request by neilschark**

### CODE REVIEW

**Review by andresterba — CHANGES_REQUESTED** [review]
Nice work!

File: .github/workflows/build-latest-docker-image.yaml [hunk 1]
```diff
@@ -24,5 +24,7 @@ jobs:
         run: docker buildx create --name arm-x86-multi-builder
       - name: Use buildx builder
         run: docker buildx use arm-x86-multi-builder
-      - name: Build Docker image
+      - name: Build Docker image latest
```
[the remark this answers is not in the document]
Comment (neilschark) [reply .github/workflows/build-latest-docker-image.yaml:27]: AFAIK every project has a latest tag on a docker image. Nextcloud is the only one I ever used which can break if you just use latest.

File: .github/workflows/build-latest-testing-docker-image.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name: build latest testing docker image
+
+on:
+  push:
+    branches:
+      - main
+
+
+jobs:
+  build-latest-testing-image-multiarch:
```
[the remark this answers is not in the document]
Comment (neilschark) [reply .github/workflows/build-latest-testing-docker-image.yaml:10]: Yes, the job name is incorrect. I will change it

File: .github/workflows/build-latest-docker-image.yaml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 name: build latest docker image
```
[the remark this answers is not in the document]
Comment (neilschark) [reply .github/workflows/build-latest-docker-image.yaml]: Good Idea, I will change it.

File: .github/workflows/build-latest-testing-docker-image.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (neilschark) [reply .github/workflows/build-latest-testing-docker-image.yaml]: Done

File: .github/workflows/build-latest-docker-image.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (neilschark) [reply .github/workflows/build-latest-docker-image.yaml]: Done

File: .github/workflows/build-latest-docker-image.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (neilschark) [reply .github/workflows/build-latest-docker-image.yaml]: I would just make sure that our project updates nicely

[the remark this answers is not in the document]
Comment (andresterba) [reply .github/workflows/build-latest-docker-image.yaml]: > I would just make sure that our project updates nicely

Maybe we should talk with @theErikss0n about versioning.
I would prefer basic SemVer and with that we can break with major releases.
But let's go with the latest tag for now until we have a real release canidate.

**Review by andresterba — APPROVED** [review]
LGTM :tada:

**Pull request merged**